### ISSUE TITLE
copying a descriptor

### ISSUE THREAD

**Issue by Gromily4** [Xerbo/furaffinity-dl#62]
is it possible to create as metadata a descriptor from <div class="submission-description user-submitted-links">

**Comment by Radiquum** [comment]
it's possible, but for what reason you need it, if it just a whole description anyways?

**Comment by Gromily4** [comment]
Since there are large amounts of text in this description. For example there is written some story. and I would like to have a full description of it, not something neutered.
Examples:
https://www.furaffinity.net/view/45866233/
https://www.furaffinity.net/view/45597683

**Comment by Radiquum** [comment]
it is full, just new lines are replaced with \n, because JSON can't handle it
<details>

![image](https://user-images.githubusercontent.com/68120017/174393857-f1f7a097-d85d-42e9-9593-de49ba8eeafc.png)

![image](https://user-images.githubusercontent.com/68120017/174393924-6586db4b-c1d9-4df7-a970-51235189a58e.png)
</details>

**Comment by Radiquum** [comment]
this is as close as I could get it to work, without /n, also it's a list, not just a string:
<details>

![image](https://user-images.githubusercontent.com/68120017/174404872-8ad9444a-fa35-4d58-bd86-d9c797ce586e.png)

![image](https://user-images.githubusercontent.com/68120017/174404945-be8961e5-4b90-4917-ac69-f75bb776b94a.png)

</details>

**Comment by Gromily4** [comment]
Yes, you should, but what do you have to do to achieve this result?

**Comment by Radiquum** [comment]
> Yes, you should, but what do you have to do to achieve this result?

it's currently implemented in my fork of python version of this script, you just need to pass some arguments to it

p.s. sorry for the slow reply, I didn't saw that you edited your comment

**Comment by Gromily4** [comment]
Thanks for your help. I downloaded your version in python, but how do I load this descripthen, since --json-description does not work.

**Comment by Radiquum** [comment]
> Thanks for your help. I downloaded your version in python, but how do I load this descripthen, since --json-description does not work.

python furaffinity-dl.py \<username\> \<category\> -m --json-description

-m needed for enabling metadata downloading
--json-description needed for download description as a list

if it's giving some kind of an error then please open an new issue on page of the fork

**Comment by Gromily4** [comment]
How do I get rid of these messages?
<i> post On the Auction Block was filtered and will not be downloaded - https://www.furaffinity.net/view/37863158/

**Comment by Deer-Spangle** [comment]
> if it's giving some kind of an error then please open an new issue on page of the fork

Just a heads-up from a nosy observer, but issues are not enabled on [your fork](https://github.com/Radiquum/furaffinity-dl) (they're disabled by default on forks)
Should be able to turn them on under `Settings > General > Features > Issues`

**Comment by Radiquum** [comment]
> issues are not enabled on [your fork](https://github.com/Radiquum/furaffinity-dl)

oops.. didn't notice that, well at least the are enabled now, thanks for your observation 😅
